src/main/java/uk/gov/pay/publicauth/healthcheck/DatabaseHealthCheck.java

```diff
@@ -0,0 +1,107 @@
+package uk.gov.pay.publicauth.healthcheck;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.health.HealthCheck;
+import io.dropwizard.setup.Environment;
+import uk.gov.pay.publicauth.app.config.PublicAuthConfiguration;
+
+import javax.inject.Inject;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import java.sql.DriverManager;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DatabaseHealthCheck extends HealthCheck {
+
+    private PublicAuthConfiguration configuration;
+
+
+
+    private static final Map<String,Long> longDatabaseStatsMap;
+    private static final Map<String,Double> doubleDatabaseStatsMap;
+
+    private Integer statsHealthy= 0;
```

Comment:
space after the equals - doesn't look like java has implicit bool to int conversion otherwise would have suggested a bool here.